src/components/Sections/SectionsContext.tsx

```diff
@@ -74,12 +199,11 @@ export const SectionsProvider = ({
         ungroupedSections,
         groupedSections,
         displayTutorialTip: areAllCategoriesPristine,
-        isSuggested: (slug: string): boolean => {
-          return suggestedKonnectors.some(konnector => konnector.slug === slug)
-        },
-        isRunning: (slug: string): boolean => {
-          return runningKonnectors.includes(slug)
-        }
+        isSuggested: (slug: string): boolean =>
+          suggestedKonnectors.some(konnector => konnector.slug === slug),
```

Comment:
Why did you use an `.some` here instead `.includes` like below ? Is there a performance issue here ?